applications/app/views/fragments/mediaBody.scala.html

```diff
@@ -98,19 +99,34 @@
                                 }
                                 </div>
 
-                                <div class="content__main-column__body" data-component="body">
+                                @if(!page.media.tags.isVerticalVideo) {
```

Comment:
non-blocking but could we use an if else rather than two ifs? This way we can say @if(page.media.tags.isVerticalVideo) {...} else {...} and not worry about checking if the tag isnt there